webnn-ops.md

```diff
@@ -0,0 +1,146 @@
+| <span class="spec">[floor <sup>element-wise unary</sup>](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-unary)</span> | <span class="x_cr">🔍</span> floor | <span class="ep_wip">🚀</span> Floor |
+| <span class="spec">[neg <sup>element-wise unary</sup>](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-unary)</span> | <span class="x_cr">🔍</span> negate | <span class="ep_wip">🚀</span> Neg |
+| <span class="spec">[elu](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-elu)</span> | <span class="x_s">✅</span> elu | <span class="ep_wip">🚀</span> Elu |
+| <span class="spec">[gemm](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-gemm)</span> | <span class="x_s">✅</span> fully_connected | <span class="ep_s">✅</span> Gemm |
+| <span class="spec">[hardSwish](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-hard-swish)</span> | <span class="x_s">✅</span> hardswish | <span class="ep_wip">🚀</span> HardSwish |
+| <span class="spec">[leakyRelu](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-leakyrelu)</span> | <span class="x_s">✅</span> leaky_relu | <span class="ep_s">✅</span> LeakyRelu |
+| <span class="spec">[pad](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-pad)</span> | <span class="x_s">✅</span> static_constant_pad | <span class="ep_wip">🚀</span> Pad |
+| <span class="spec">[averagePool2d <sup>pooling</sup>](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-pool2d)</span> | <span class="x_s">✅</span> average_pooling_2d | <span class="ep_s">✅</span> GlobalAveragePool |
+| ^^ | ^^ | <span class="ep_s">✅</span> AveragePool |
+| <span class="spec">[maxPool2d <sup>pooling</sup>](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-pool2d)</span> | <span class="x_s">✅</span> max_pooling_2d | <span class="ep_s">✅</span> GlobalMaxPool |
+| ^^ | ^^ | <span class="ep_s">✅</span> MaxPool |
+| <span class="spec">[prelu](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-prelu)</span> | <span class="x_s">✅</span> prelu | <span class="ep_wip">🚀</span> Prelu |
+| <span class="spec">[relu](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-relu)</span> | <span class="x_s">✅</span> clamp | <span class="ep_s">✅</span> Relu |
+| <span class="spec">[resample2d](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-resample2d)</span> | <span class="x_s">✅</span> static_resize_bilinear_2d | <span class="ep_s">✅</span> Resize |
+| <span class="spec">[reshape](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-reshape)</span> | <span class="x_s">✅</span> static_reshape | <span class="ep_s">✅</span> Reshape |
+| <span class="spec">[sigmoid](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-sigmoid)</span> | <span class="x_s">✅</span> sigmoid | <span class="ep_s">✅</span> Sigmoid |
+| <span class="spec">[split](https://www.w3.org/TR/webnn/#api-mlgraphbuilder-split)</span> | <span class="x_cr">🔍</span> even_split | <span class="ep_wip">🚀</span> Split |
```

Comment:
WebNN XNNPack backend will support even split by even_split2, even_split3 and even_split4, support uneven split by static_slice.

Reply:
Thanks @miaobin , fixed.

Reply:
LGTM, thanks!